_examples/signals-to-json.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+/* Example usage, showing systemd unit property changes:
```

Comment:
This syntax does not follow go convention. I suggest you use forward-slashes: https://golang.org/doc/effective_go.html#commentary